sdk/core/azure-core/src/main/resources/META-INF/native-image/com.azure/azure-core/native-image.properties

```diff
@@ -19,6 +19,7 @@ Args=\
                              com.azure.core.implementation.logging.DefaultLogger,\
                              com.azure.core.test.TestBase,\
                              com.azure.core.util.Configuration,\
+                             com.azure.core.util.logging.ClientLogger,\
```

Comment:
You guys really haven't thought this through. You're targeting your settings towards Spring Boot apps, and making it impossible to use your libraries via native image, in other configurations.